doc/sphinx/practical-tools/coqide.rst

```diff
@@ -62,8 +62,8 @@ In the figure :ref:`CoqIDE main screen <coqide_mainscreen>`,
 the running buffer is `Fermat.v`, all commands until
 the ``Theorem`` have been already executed, and the user tried to go
 forward executing ``Induction n``. That command failed because no such
-tactic exists (tactics are now in lowercase…), and the wrong word is
-underlined.
+tactic exists (names of standard tactics are writen in lowercase with
+words separated by underscores), and the failing command is underlined.
```

Comment:
The underscore part is incorrect, since there are standard tactics (eg `simple apply`) that does not use `_` as a separator but rather ` `.

Reply:
I removed the mention of underscores.